Frontend/src/GoogleMap/GoogleMap.js

```diff
@@ -101,7 +101,7 @@ const GoogleMap = () => {
   const closePlaceInfoBox = () => setPlaceInfoBoxVisibility(false);
 
   const handleConfirm = () => {
-    dispatch(addPlacelocationActions.changeIsSelecting(false));
+    dispatch(addPlacelocationActions.changeIsSelecting({ isSelecting: false }));
```

Comment:
Nie rozumiem tej zmiany. Nie zauważyłem żeby były jakieś zmiany co do tego kodu.

Reply:
Tutaj kiedy zatwierdzało się lokalizację to zamiast isSelecting false było undefined ponieważ w kodzie przekazywany jest obiekt zamiast samej wartości logicznej

Reply:
Teraz rozumiem